config/default_configs/default_config.yml

```diff
@@ -123,9 +123,6 @@ reference_job_id:
   help: |-
     Identifier of job to use as the "reference" solution in the quicklook plot; the current job's results get compared to the results of the quicklook job on the main branch
   value: ~
-post_process:
```

Comment:
@Sbozzolo just want to double check we don't use this anymore.

Reply:
I didn't even know it was a thing. Yes, I don't think it is used anywhere now.